test/bench/bench.cpp

```diff
@@ -132,12 +157,20 @@ void load_benchmark(const fs::path& path, const std::string& name_prefix)
     const auto wasm_binary = std::make_shared<const fizzy::bytes>(
         std::istreambuf_iterator<char>{wasm_file}, std::istreambuf_iterator<char>{});
 
-    register_benchmark("parse/" + base_name,
-        [wasm_binary](benchmark::State& state) { benchmark_parse(state, *wasm_binary); });
-
-    register_benchmark("instantiate/" + base_name,
-        [wasm_binary](benchmark::State& state) { benchmark_instantiate(state, *wasm_binary); });
+    for (const auto entry : engine_registry)  // Register parse benchmarks.
```

Comment:
Could be reference, but perhaps not big difference
```suggestion
    for (const auto& entry : engine_registry)  // Register parse benchmarks.
```